src/main/java/core/basesyntax/strategy/PurchaseOperation.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Inventory;
+
+public class PurchaseOperation implements FruitOperationHandler {
+    @Override
+    public void executeOperation(FruitTransaction transaction, Inventory inventory) {
```

Comment:
According to the checklist, we should avoid beginning method implementation with an empty line. Remove the empty line after the method declaration.